main.go

```diff
@@ -153,11 +153,20 @@ func startLeaderElection(leaderElectionClient kubernetes.Interface, recorder rec
 	if err != nil {
 		klog.Fatalf("error getting hostname: %s", err.Error())
 	}
+	
+	kubeconfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{},
+	)
+	namespace, _, err := kubeconfig.Namespace()
+	if err != nil {
+		namespace = "submariner"
+	}
```

Comment:
I think it would be good to log Info messages indicating which namespace is being used, ie

    if err != nil {
        klog.Infof("Could not obtain a namespace to use for the leader election lock - the error was: %v. Using the default \"submariner\ "namespace.", err)
        namespace = "submariner"
    } else {
         klog.Infof("Using namespace %s for the leader election lock", namespace)
    }

Reply:
@Negashev  can you add that last change?, then we can merge it, thank you

Reply:
@mangelajo Yes, one moment